featurize_pdbbind_cluster.py

```diff
@@ -62,18 +78,23 @@ def featurize_pdbbind(pdbbind_dir, script_dir, script_template, num_jobs,
     print job_dirs
 
     # TODO(rbharath): This is horrible. Clean this script up...
-    pickle_out = os.path.join(pickle_dir, "features%d.p" % job)
+    pickle_out = os.path.join(pickle_dir, "features%d.pkl.gz" % job)
     #TODO(enf): the path needs to be user-independent 
-    command = " ".join(["python", "~/software/pbs_utils/featurize_pdbbind_cluster_job.py",
+    
+    command = " ".join(["python", "/scratch/users/enf/deep-docking/cluster_utils/featurize_pdbbind_cluster_job.py",
```

Comment:
Can you make this a flag? We should fix this now since I'm sure this will cause us headaches.